### ISSUE TITLE
Consider removing the public service from the kubernetes-dns integration test

### ISSUE THREAD

**Issue by raboof** [akka/akka-management#495]
This test currently defines a 'public' k8s service to expose the HTTP service.

While on the one hand this arguably makes the example/test more realistic/complete, on the other hand exposing the service doesn't have anything to do with the feature that is demonstrated, cluster bootstrap.

Perhaps we should remove it, removing some 'noise' from the example/test.

**Comment by chbatey** [comment]
I agree that we should either have a fully fledged tutorial with ingress or just show the barebones required by bootstrap
